test/partnership.e2e-spec.ts

```diff
@@ -23,11 +27,15 @@ import { createPartnership } from './utility/create-partnership';
 describe('Partnership e2e', () => {
   let app: TestApp;
   let project: Raw<Project>;
+  const password: string = faker.internet.password();
 
   beforeAll(async () => {
     app = await createTestApp();
     await createSession(app);
-    await createUser(app);
+    const user = await createUser(app, { password });
+    await grantPower(app, user.id, Powers.CreateOrganization);
+    await login(app, { email: user.email.value, password });
```

Comment:
Since this use case is so common it should probably be abstracted and would be an opportunity to stop using the deprecated `createUser` function. It could be called `registerUserWithPower`.